crates/red_knot_python_semantic/src/semantic_index/use_def/bitset.rs

```diff
@@ -32,17 +32,26 @@ impl<const B: usize> BitSet<B> {
         bitset
     }
 
+    #[allow(unused)]
```

Comment:
I'm not sure if we can use it already because of our minimal msrv but this is a perfect use case for the new [`[#expect]`](https://blog.rust-lang.org/2024/09/05/Rust-1.81.0.html#expectlint) attribute.